Makefile

```diff
@@ -5,6 +5,7 @@ EXAMPLE_FOLDER             ?=
 EXAMPLE_NAME               ?=
 INSTALL_EXTRAS             ?=
 VERSION                    ?=
+IS_GPU                     ?= False
```

Comment:
Please move it to line 4 to make all not-None arguments together.

Reply:
moved